.github/workflows/client.yml

```
@@ -33,23 +36,15 @@ jobs:
       fail-fast: true
     steps:
       - uses: actions/checkout@v2
-
-      - name: Setup node
-        uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
-
-      - name: Install dependencies
-        uses: bahmutov/npm-install@v1
-
+      - name: bahmutov/npm-install@v1
```

Should this be `uses:`?
```suggestion
      - uses: bahmutov/npm-install@v1
```

---

YES!